### PR TITLE
Error entity type (wide data)

### PR DESCRIPTION
**Pull request by dfalster**
Solves first issue raised in #725, that entity_type and value_type are not being successfully read from a column of data.

### CODE REVIEW

**Comment by dfalster** [comment]
@ehwenk I suggest we address the other part of #725 issue (for long data) via a seperate branch and PR

**Comment by ehwenk** [comment]
> @ehwenk I suggest we address the other part of #725 issue (for long data) via a seperate branch and PR

sure.

**Review by ehwenk — APPROVED** [review]
I've confirmed that this fixes the pre-existing problem. 

There are two studies with entity types not being read in (entity type = NA). These were fixed last week on this branch.

> `https://github.com/traitecoevo/austraits.build/tree/fix-entity_type-value_type-units-not-reading-in`

This branch also fixes, for now, various problems with value_type.

I'm not sure if it is better to redo those changes here before merging, or wait until this branch is merged in and have those as a separate commit.

File: R/process.R
Comment (dfalster) [reply R/process.R:1071]: https://stackoverflow.com/a/6559049

**Comment by dfalster** [comment]
Hi Lizzy,

Sorry but I don't understand what you were asking in your last message. You refer several times to "this" branch, but I suspect some of the this actually meant the one with the link?

In any case, perhaps two seperate commits is best?

**Comment by ehwenk** [comment]
Two commits is fine. There is a second branch where I made changes to metadata files and which had the initial changes for fixing entity_type that are now superceded. The other branch is: https://github.com/traitecoevo/austraits.build/tree/fix-entity_type-value_type-units-not-reading-in  .

**Pull request merged**
